server/server.go

```diff
@@ -50,6 +50,8 @@ var (
 	// when the server sits behind a reverse proxy with a prefix (i.e. /tegola)
 	URIPrefix = "/"
 
+	ProxyProtocol string
```

Comment:
Reading the PR the option is self-explanatory. Without context it is not. A comment would be nice here - or a ref to this PR at least. 🙏

Reply:
Oupsi daisy !
It would indeed be better to explain this param. I've just added the comment :)